src/libmongoc/src/mongoc/mongoc-database-private.h

```diff
@@ -43,7 +44,8 @@ _mongoc_database_new (mongoc_client_t *client,
                       const char *name,
                       const mongoc_read_prefs_t *read_prefs,
                       const mongoc_read_concern_t *read_concern,
-                      const mongoc_write_concern_t *write_concern);
+                      const mongoc_write_concern_t *write_concern,
+		      int64_t timeout_ms);
```

Comment:
clang-format?

Reply:
Yes, same weirdness with my editor, I'll do a final clang-format before committing this PR.